### ISSUE TITLE
Clicking the 'quote' button on a single link crashes the editor

### ISSUE THREAD

**Issue by jeromesimeon** [accordproject/markdown-editor#239]
**Describe the bug**
Creating a quote over a paragraph containing a single link crashes the editor.

**To Reproduce**
1. Create a paragraph with some text
2. Click on the paragraph and create a 'link'
3. Click on 'quote'
4. The editor crashes

**Expected behavior**
The editor should display a quote containing the link

**Comment by jeromesimeon** [comment]
The error trace from Chrome:
```
Uncaught (in promise) TypeError: Cannot read property 'nodeRefs' of undefined
    at search (slate-react.es.js:2185)
    at search (slate-react.es.js:2186)
    at search (slate-react.es.js:2186)
    at findDOMNode (slate-react.es.js:2190)
    at onQuery (slate.es.js:9240)
    at next (slate.es.js:11164)
    at onQuery (slate.es.js:9239)
    at next (slate.es.js:11164)
    at Editor.run (slate.es.js:11198)
    at Editor.query (slate.es.js:11062)
```

**Comment by raghav4** [comment]
Hi @jeromesimeon, it must have been fixed by now as I tried recreating the issue with whatever you've suggested but couldn't encounter it, here's the link to the video reference https://www.loom.com/share/89b49297903543e7a37572cd8f2323df

Thank you

**Comment by irmerk** [comment]
@jeromesimeon this was fixed in [markdown-transform 0.9.10](https://github.com/accordproject/markdown-transform/releases/tag/v0.9.10) correct?

**Comment by jeromesimeon** [comment]
@irmerk I think so! Closing this issue.
